### PR TITLE
change OpenShift references to Origin in readme

### PR DESCRIPTION
**Pull request by pweil-**
change OpenShift references to Origin in readme

@smarterclayton

### CODE REVIEW

**Comment by pweil-** [comment]
ref: https://github.com/openshift/origin/pull/3958#issuecomment-127378087

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-OpenShift Application Platform
+Origin Application Platform
```
Comment (smarterclayton) [inline README.md]: Don't change - I think as the H1 we need to identify OpenShift first (community brand).  Origin is the name of the community technology.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 ==============================
 
 [![GoDoc](https://godoc.org/github.com/openshift/origin?status.png)](https://godoc.org/github.com/openshift/origin)
 [![Travis](https://travis-ci.org/openshift/origin.svg?branch=master)](https://travis-ci.org/openshift/origin)
 
-This is the source repository for [OpenShift 3](https://openshift.github.io), based on top of [Docker](https://www.docker.io) containers and the
+This is the source repository for [Origin](https://openshift.github.io), based on top of [Docker](https://www.docker.io) containers and the
```
Comment (smarterclayton) [inline README.md]: Can't change - but you can add a separate sentence. "Origin is a distribution of Kubernetes optimized for enterprise application development and deployment, used by OpenShift and Atomic Enterprise".

File: README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 [Kubernetes](https://github.com/GoogleCloudPlatform/kubernetes) container cluster manager.
-OpenShift adds developer and operational centric tools on top of Kubernetes to enable rapid application development,
+Origin adds developer and operational centric tools on top of Kubernetes to enable rapid application development,
```
Comment (smarterclayton) [inline README.md]: With the previous sentence added you can leave this as Origin.

File: README.md [hunk 4]
```diff
@@ -23,19 +23,19 @@ easy deployment and scaling, and long-term lifecycle maintenance for small and l
...
-* **[OpenShift Public Documentation](http://docs.openshift.org/latest/welcome/index.html)**
+* **[Origin Public Documentation](http://docs.openshift.org/latest/welcome/index.html)**
 * The **[Trello Roadmap](https://ci.openshift.redhat.com/roadmap_overview.html)** covers the epics and stories being worked on (click through to individual items)
 * **[Technical Architecture Presentation](https://docs.google.com/presentation/d/1Isp5UeQZTo3gh6e59FMYmMs_V9QIQeBelmbyHIJ1H_g/pub?start=false&loop=false&delayms=3000)**
 * **[System Architecture](https://github.com/openshift/openshift-pep/blob/master/openshift-pep-013-openshift-3.md)** design document
 
 For questions or feedback, reach us on [IRC on #openshift-dev](https://botbot.me/freenode/openshift-dev/) on Freenode or post to our [mailing list](https://lists.openshift.redhat.com/openshiftmm/listinfo/dev).
 
-NOTE: OpenShift Origin 1.0 has been released [releases page](https://github.com/openshift/origin/releases). Feedback, suggestions, and testing are all welcome!
+NOTE: Origin 1.0 has been released [releases page](https://github.com/openshift/origin/releases). Feedback, suggestions, and testing are all welcome!
```
Comment (smarterclayton) [inline README.md]: This is probably fine

File: README.md [hunk 5]
```diff
@@ -60,14 +61,14 @@ The easiest way to run OpenShift Origin is in a Docker container (OpenShift requ
...
-and starts and stops Docker containers, mounts remote volumes, and monitors the system (/sys) to report performance and health info. You can strip all of these options off and OpenShift will still start, but you won't be able to run pods (which is kind of the point).
+**Security!** Why do we need to mount your host, run privileged, and get access to your Docker directory? Origin runs as a host agent (like Docker)
+and starts and stops Docker containers, mounts remote volumes, and monitors the system (/sys) to report performance and health info. You can strip all of these options off and Origin will still start, but you won't be able to run pods (which is kind of the point).
 
 Once the container is started, you can jump into a console inside the container and run the CLI.
 
     $ sudo docker exec -it origin bash
 
-    # Start the OpenShift integrated registry in a container
+    # Start the Origin integrated registry in a container
```
Comment (smarterclayton) [inline README.md]: Just say integrated registry

File: README.md [hunk 6]
```diff
@@ -23,19 +23,19 @@ easy deployment and scaling, and long-term lifecycle maintenance for small and l
 
 **Learn More:**
 
-* **[OpenShift Public Documentation](http://docs.openshift.org/latest/welcome/index.html)**
+* **[Origin Public Documentation](http://docs.openshift.org/latest/welcome/index.html)**
```
Comment (smarterclayton) [inline README.md]: Just Public Documentation

File: README.md (hunk 2 above)
Comment (smarterclayton) [inline README.md]: For the link text use "OpenShift Origin" instead of "OpenShift 3"

Comment (smarterclayton) [inline README.md]: Also, use "OpenShift 3" in my suggestion text

File: README.md [hunk 7]
```diff
@@ -132,19 +133,27 @@ The API documentation can be found [here](http://docs.openshift.org/latest/rest_
...
-    Kubernetes today is focused around composing containerized applications - OpenShift adds
+    Kubernetes today is focused around composing containerized applications - Origin adds
     building images, managing them, and integrating them into deployment flows.  Our goal is to do
-    most of that work upstream, with integration and final packaging occurring in OpenShift.  As we
+    most of that work upstream, with integration and final packaging occurring in Origin.  As we
     iterate through the next few months, you'll see this repository focus more on integration and
     plugins, with more and more features becoming part of Kubernetes.
 
-2. What can I run on OpenShift?
+2. How does Atomic Enterprise relate to Origin and OpenShift?
```
Comment (smarterclayton) [inline README.md]: This paragraph works, but the previous one is weak.  It highlights how this naming setup is confused (origin as a technology) and distinct from product.  I would make the previous paragraph about technology (enterprise distribution of Kube), and this is just a paragraph about product - describe AEP and OpenShift as two products that exist on top of the technology.  Don't include source to image here.

**Comment by danmcp** [comment]
@pweil- Bump

**Comment by pweil-** [comment]
@smarterclayton - round 2 on the readme

**Comment by smarterclayton** [comment]
@liggitt as well

On Aug 11, 2015, at 9:53 AM, Paul Weil <notifications@github.com> wrote:

@smarterclayton <https://github.com/smarterclayton> - round 2 on the readme

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3990#issuecomment-129882248>.

**Comment by pweil-** [comment]
rebased

**Comment by pweil-** [comment]
rebased again.  Any more comments on this?

**Comment by smarterclayton** [comment]
I think this is ok.  [merge]

**Pull request merged**